lib/crowbar/init/helpers.rb

```diff
@@ -208,9 +208,8 @@ def api_constraint(*versions)
           break if versions_requested.nil?
           return true if versions_requested[:major].to_i == major &&
               versions_requested[:minor].to_i <= minor
+          halt 406, { "Content-Type" => "application/vnd.crowbar.v#{major}.#{minor}+json" }, ""
```

Comment:
not sure about having `halt` within the block, it will stop the iteration after the first round, is this the expected behaviour?